sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -22,6 +22,7 @@ export const DATE_ONLY_ISO_FORMAT = "YYYY-MM-DD";
 export const DATE_ONLY_FORMAT = "MMM DD YYYY";
 export const DATE_TIME_FORMAT = "YYYY-MM-DD HH:mm:ss";
 export const TIMESTAMP_CONTINUOUS_FORMAT = "YYYY-MM-DD_HH.mm.ss";
+export const CUSTOMIZED_TIMESTAMP_CONTINUOUS_FORMAT = "YYYYMMDD-HHmmssSSS";
```

Comment:
Just as a minor suggestion, I would call it `TIMESTAMP_CONTINUOUS_EXTENDED_FORMAT` since it is similar to `TIMESTAMP_CONTINUOUS_FORMAT`.